tests/e2e/specs/out-of-game/stats.spec.js

```diff
@@ -268,42 +267,69 @@ describe('Usage stats', () => {
   });
 
   it('Sends the counts of games played and unique players for each week of each season', () => {
-    cy.request('http://localhost:1337/stats').then(({body: seasons}) => {
-      // Clubs 2022 stats
-      const clubs2022 = seasons.find(({name}) => name === 'Clubs 2022');
-      expect(clubs2022).not.to.be.undefined;
-      // Week 1 stats
-      expect(clubs2022.gameCounts[0]).to.eq(4);
-      expect(clubs2022.uniquePlayersPerWeek[0]).to.eq(3);
-      // Week 2 stats
-      expect(clubs2022.gameCounts[1]).to.eq(2);
-      expect(clubs2022.uniquePlayersPerWeek[1]).to.eq(4);
-      // Week 3 stats
-      expect(clubs2022.gameCounts[2]).to.eq(0);
-      expect(clubs2022.uniquePlayersPerWeek[2]).to.eq(0);
-      // Week 3 stats
-      expect(clubs2022.gameCounts[3]).to.eq(1);
-      expect(clubs2022.uniquePlayersPerWeek[3]).to.eq(2);
+    cy.request('http://localhost:1337/stats/current').then(({ body: seasons }) => {
+      const populateSeason = (season) => {
+        return new Cypress.Promise((resolve, reject) => {
```

Comment:
I was having trouble with just using cy.request, I was getting $chainer as a return, I think it's because It doesn't resolve the same as a promise inside the cypress.promise.all